tests/scripts/run_performance.sh

```diff
@@ -17,6 +17,8 @@ run_perf_models_other() {
 
     if [ "$tt_arch" == "wormhole_b0" ]; then
         env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/resnet50/tests/test_perf_e2e_resnet50.py -m $test_marker
+
+        env WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/distilbert/tests/test_perf_distilbert.py -m $test_marker
```

Comment:
Referring to the [document](https://github.com/tenstorrent/tt-metal/blob/53c32c0c0da926f97bd0eb042e70fd54c2866f44/docs/source/ttnn/ttnn/demos.rst#L31:~:text=Please%20add%20your%20tests%20to%20either%20run_perf_models_other()%2C%20run_perf_models_llm_javelin()%2C%20or%20run_perf_models_cnn_javelin()%20within%20run_performance.sh.%20Only%20tests%20marked%20with%20%40pytest.mark.models_performance_bare_metal%20will%20be%20included.), the pytest fixture `@pytest.mark.models_performance_bare_metal` has been added only to `test_performance_distilbert_for_qa`, so it will consider only `test_performance_distilbert_for_qa` and ignore `test_distilbert_perf_device`